lib/index.js

```diff
@@ -95,21 +95,24 @@ class Parser extends Transform {
       }
     }
     // Normalize option `delimiter`
-    if(options.delimiter === undefined || options.delimiter === null || options.delimiter === false){
-      options.delimiter = Buffer.from(',')
-    }else if(typeof options.delimiter === 'string' && options.delimiter.length !== 0){
-      options.delimiter = Buffer.from(options.delimiter)
-    }else if(
-      (Buffer.isBuffer(options.delimiter) && options.delimiter.length === 0) ||
-      (typeof options.delimiter === 'string' && options.delimiter.length === 0) ||
-      (!Buffer.isBuffer(options.delimiter) && typeof options.delimiter !== 'string')
-    ){
-      throw new CsvError('CSV_INVALID_OPTION_DELIMITER', [
-        'Invalid option delimiter:',
-        'delimiter must be a non empty string or buffer,',
-        `got ${JSON.stringify(options.delimiter)}`
-      ])
-    }
+    let delimiter = options.delimiter
```

Comment:
Let's use options.delimiter everwhere, there is no need to confuse the code reader with a new variable which is just the same as another one.

Reply:
alright